blocks/primlancers/primlancers.css

```diff
@@ -0,0 +1,208 @@
+.primlancers {
+    padding: 80px 0 60px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.primlancers__title {
+    margin: 0;
+    font-family: "Roboto Condensed", Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 34px;
+    line-height: 40px;
+    text-align: center;
+    letter-spacing: -0.01em;
+    text-transform: uppercase;
+    color: #683d83;
+}
+
+.title_black {
+    color: #000000;
+}
+
+.primlancers__text {
+    font-family: "Roboto Condensed", Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 34px;
+    line-height: 40px;
+    text-align: center;
+    letter-spacing: -0.01em;
+    text-transform: uppercase;
+    color: #000000;
+    margin: 40px 0 0;
+    z-index: 1;
+}
+.primlancers__video-container {
+    position: relative;
```

Comment:
The border radius now is set elsewhere and the top margin is too. The overlay bleeds out because of it. Is it possible to put the margin on the text (use the pre-made "heading" block) and use this video container to set the border-radius and control overflow?

Reply:
Thank you. I've put margin on text instead. However, For now I can't find the way to set border radius of absolute item from outer div yet. So have to put border-radius on every layer for now.